templates/README.md

```diff
@@ -20,64 +20,77 @@ Only the structure of the project file is relevant to the template. The actual
 list of source code and targets isn't.
 
 We currently have template files for GNU Make, Visual Studio 2010 to 2015,
-and Bazel. In the future, we would like to expand to generating gyp or cmake
+and [Bazel](http://bazel.io). In the future, we would like to expand to
+generating [gyp](https://gyp.gsrc.io/) or [cmake](https://cmake.org)
```

Comment:
Only for Node. I'm not sure if gyp for C++ is desirable. I suspect not, but
back in they day we thought it might be.

On Tue, Mar 1, 2016, 4:54 PM Nicolas Noble <notifications@github.com> wrote:

> In templates/README.md
> <https://github.com/grpc/grpc/pull/5524#discussion_r54663529>:
>
> > @@ -20,64 +20,77 @@ Only the structure of the project file is relevant to the template. The actual
> >  list of source code and targets isn't.
> >
> >  We currently have template files for GNU Make, Visual Studio 2010 to 2015,
> > -and Bazel. In the future, we would like to expand to generating gyp or cmake
> > +and [Bazel](http://bazel.io). In the future, we would like to expand to
> > +generating [gyp](https://gyp.gsrc.io/) or [cmake](https://cmake.org)
>
> We ARE generating gyp...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/5524/files#r54663529>.
>